src/config.ts

```diff
@@ -246,8 +251,7 @@ export async function getConfig(): Promise<Config> {
   if (metamask) {
     // If Metamask is detected, we use the network configured there.
     const ready = await checkMetaMask(metamask);
-    if (ready === null) throw new Error("Not able to connect to Metamask, check for multiple Web3 providers");
-    network = ready;
+    if (ready) network = ready;
```

Comment:
Ok this is maybe a weird one, I started having issues without having Metamask installed just with Brave Wallets that this error has been thrown around..
So IMO we should be fine if we just check for `ready` and if something returns we can overwrite `network` and else we stay on `homestead`